Source/msg.cpp

```diff
@@ -34,6 +34,9 @@
 #include "objects.h"
 #include "options.h"
 #include "pfile.h"
+#ifdef _DEBUG
+#include "playerdat.hpp"
+#endif

```

Comment:
Is this include still needed?

Reply:
Nope, should've removed that in a previous rebase...